src/Mutts.jl

```diff
@@ -73,34 +73,44 @@ actions right before an object is branched.
 branchactions(obj) = nothing
 
 """
-    markimmutable!(obj)
+    mark_immutable!(obj)
 
 Freeeze `obj` from further mutations, making it eligible to pass to
 other Tasks, branch! from it, or otherwise share it.
```

Comment:
```suggestion
Freeze `obj` from further mutations, making it eligible to pass to
other Tasks, branch! from it, or otherwise share it.
```
Hehe just noticed this typo